LedRGB.cpp

```diff
@@ -0,0 +1,37 @@
+#include "LedRGB.h"
+#include "Led.h"
+
+LedRGB::LedRGB() : RedLed(255), GreenLed(255), BlueLed(255) {
+}
+
+LedRGB::LedRGB(unsigned redPin, unsigned greenPin, unsigned bluePin)
+    : RedLed(redPin), GreenLed(greenPin), BlueLed(bluePin) {
+}
+
+void LedRGB::setRGB(unsigned r, unsigned g, unsigned b) {
+    redBrightness = r / 2.55;
+    greenBrightness = g / 2.55;
+    blueBrightness = b / 2.55;
+    RedLed.setBrightness(redBrightness * brightness);
+    GreenLed.setBrightness(greenBrightness * brightness);
+    BlueLed.setBrightness(blueBrightness * brightness);
+}
+
+void LedRGB::setBrightness(unsigned percent) {
+    brightness = percent / 100;
```

Comment:
2 things wrong with this line.
1. brightness is an integer not a float
2. your are performing integer division so
1/100 == 0
50/100 ==0
99/100==0
100/100==1
50/100.0=50.0